src/v1/error.js

```diff
@@ -44,6 +44,12 @@ class Neo4jError extends Error {
     super( message );
     this.message = message;
     this.code = code;
+    Object.defineProperty(this, 'name', {
```

Comment:
Yeah, code sets `enumerable: false`. All other properties are enumerable. Can we have `name` similar?